ush/parsing_namelists_CICE.sh

```diff
@@ -388,11 +209,11 @@ cat > ice_in <<eof
    f_hi            = 'mdhxx'
    f_hs            = 'mdhxx' 
    f_Tsfc          = 'mdhxx' 
-   f_sice          = 'x' 
+   f_sice          = 'mdhxx' 
```

Comment:
@DeniseWorthen @LydiaStefanova-NOAA Were these fields we had specifically turned off in the workflow?  Should we keep these off in the workflow or turn them back on?

Reply:
I can't answer this - I don't know what these settings do, nor what they should be. Noticed in the configuration spreadsheet that a number of them were changed from 'mdhxx' to 'x' between P3.1 and P4, but don't know why.

Reply:
@LydiaStefanova-NOAA this is for which fields we want in the ice history output files.  'x' means it's not included and then 'mdhxx' means we will have output for this.   These were turned off as you mentioned between P3.1 and P4 for the workflow, but have remained in the regression tests as its helpful for debugging.

Reply:
There were a number of these types of changes. At first I just changed the ones that seem to have been updated since the last commit, then I noticed other settings also didn't match an naively updated them. We can revert 2dcf8f2b, which is the one where I made that second batch of changes (and then re-do the one mispaste of the new sw_redist setting that was also fixed there). Take a look at the changes from just that commit and let me know.

Reply:
@WalterKolczynski-NOAA it's a lot easier when these files just match up exactly.  I do want to say this saved us a fair amount of space on benchmark runs though.  @LydiaStefanova-NOAA I guess the question is, do we want the extra output for ice files in the full 168 runs, or do we want to keep the list that was trimmed down in P4 for ice?  Should we check with @DeniseWorthen Sulagna and @rgrumbine and others looking at ice output to see if we can continue without the extra ice output or if it'd be good to add these back -- if for nothing else to have more consistency with the rt system.

Reply:
I went through the ice.nml section to list what's different from P6 - see spreadsheet [here](https://docs.google.com/spreadsheets/d/1CpJ7U33wIA-uWlqFewgBNbQ8AtAw0727d9c49QMiHnM/edit?usp=sharing). Red=newly turned on; Blue=newly turned off.

I went by [https://cesmcice.readthedocs.io/en/latest/users_guide/ice_history.html](url) to tie the parameters to descriptions. A couple of those were not on the list (f_dsnow and f_bot). 

Of the newly turned off fields (8), we definitely want f_mlt_onset and f_frz_onset back on. Maybe f_divu and f_shear too. 

For the newly turned on fields (20) - most of them relate to radiation/albedo, salt, snow, ocean currents, atm winds. They may be useful to someone for debugging/sanity checking - I will leave it to others to weigh on that.

Reply:
Per later comment from Jessica, all those that were turned off have been turned back on.